src/helix-ui/styles/components/tabs.less

```diff
@@ -1,25 +1,19 @@
+//* ===== LAYOUT & APPEARANCE =============== */
 hx-tab {
-  background-color: transparent;
   border-radius: 2px 2px 0 0;
   border: 1px solid transparent;
-  bottom: -1px;
   color: @gray-700;
   cursor: pointer;
-  display: inline-flex;
-  flex-shrink: 0;
   font-size: 0.875rem;
   font-weight: 500;
-  padding: 0.5rem 0.75rem;
-  position: relative;
-  z-index: 0;
 
   &:hover {
     color: @cyan-500;
   }
 
-  &:active {
+  &:focus {
     color: @cyan-700;
-    z-index: 1;
+    z-index: 1; // keeps focus ring in front of adjacent tabs
```

Comment:
We should consider `z-index` regions/elevations at some point, rather than hard-coded values.

An example from Material Design: https://material.io/design/environment/elevation.html?image=https%3A%2F%2Fstorage.googleapis.com%2Fspec-host-backup%2Fmio-design%252Fassets%252F0B6xUSjjSulxceF9udnA4Sk5tdU0%252Fbaselineelevation-chart.png#default-elevations